plugins/zoom-to-fit/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "outDir": "build",
+    "target": "es6",
+    "allowJs": true,
+    "sourceMap": true,
+    "moduleResolution": "nodenext",
+    "lib": ["es2021", "dom"]
+  },
+  "include": [
+    "src", "test"
+  ],
+}
```

Comment:
This does not match other tsconfigs in samples, (in particular the `target`, `module`, and `moduleResolution`). Not sure what it is supposed to look like though.

Reply:
It's the same as what I used for toolbox-search, although I don't remember exactly how I arrived at that.

Reply:
Ok, could you file a bug for checking on this and updating the configs to be consistent?

Reply:
Done, #1871.